src/index.ts

```diff
@@ -566,7 +574,7 @@ module.exports.init_markdown_it = function (md, markdownitSub, markdownitSup,
         token: state.tokens.length - 1,
 
         // Token level.
-        //
+        // @ts-ignore
         level: state.level,
```

Comment:
markdown-it types say that `level` isn't a property here, but for now, not making any changes that affect behavior